single.php

```diff
@@ -14,6 +14,7 @@
     $local_lang = '_'.$languages[1];
 } else {
     $lang = '';
+    echo "ttit";
```

Comment:
@S-mardii @Huyeng is this intended? or is it a debug message that you forgot to remove?